scipy/linalg/__init__.py

```diff
@@ -121,8 +121,9 @@
    solve_sylvester - Solve the Sylvester matrix equation
    solve_continuous_are - Solve the continuous-time algebraic Riccati equation
    solve_discrete_are - Solve the discrete-time algebraic Riccati equation
+   solve_continuous_lyapunov - Solve the continous-time Lyapunov equation
    solve_discrete_lyapunov - Solve the discrete-time Lyapunov equation
-   solve_lyapunov - Solve the (continous-time) Lyapunov equation
```

Comment:
I think you need to keep the old name for the refguide check:

```
solve_lyapunov - A deprecated synonym for `solve_continuous_lyapunov`
```

or something along these lines.